<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjetsListForm</class>
 <widget class="QWidget" name="ObjetsListForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="styleSheet">
      <string notr="true">QSplitter {
    background-color: #ffffff; /* установите фоновый цвет сплиттера */
    border: 1px solid #000000; /* установите границы сплиттера */
    spacing: 10px; /* установите расстояние между областями сплиттера */
}</string>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>4</number>
     </property>
     <widget class="QGroupBox" name="groupBoxObjects">
      <property name="title">
       <string>Список объектов</string>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="0">
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">background-color: &quot;lightgreen&quot;;</string>
           </property>
           <property name="text">
            <string> 1-я категория</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">background-color: &quot;lightgoldenrodyellow&quot;;</string>
           </property>
           <property name="text">
            <string>2-я категория</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">background-color: &quot;azure&quot;;</string>
           </property>
           <property name="text">
            <string> 3-я категория</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">background-color: &quot;tomato&quot;;</string>
           </property>
           <property name="text">
            <string>Не работает</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="tableViewObjects">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QToolButton" name="toolButtonAddObject">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Добавить терминал...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/Images/add_plus_icon.png</normaloff>:/Images/add_plus_icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Обмен с центром</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" colspan="3" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="labelLastDeploys">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="labelDep1">
         <property name="text">
          <string>Задержки обена с ЦБ</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelDep3">
         <property name="text">
          <string>Интервал проверки каждые</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelDap2">
         <property name="text">
          <string>Отображать если нет данных более</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="spinBoxInterval">
         <property name="suffix">
          <string> мин.</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>300</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushButtonRefreshDeploys">
         <property name="text">
          <string>Обновить</string>
         </property>
         <property name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/Images/refresh_icon.png</normaloff>:/Images/refresh_icon.png</iconset>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="spinBoxPorog">
         <property name="suffix">
          <string> мин.</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1440</number>
         </property>
         <property name="value">
          <number>120</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QTableView" name="tableViewDeploys">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item row="7" column="0" colspan="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="labelWaiting">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resource.qrc">:/Images/waiting_icon_big.png</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
